spec/migrations/add_path_unique_to_harvest_items_spec.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+require_migration!
+
+describe AddPathUniqueToHarvestItems, :migration do
+  let(:longer_blob) do
+    { error: 'null', fixes: [{ version: 1, problems: { FL001: { status: 'NoOperation' } } }] }
+  end
+  let(:datetime) { "'#{Time.zone.now}'" }
+  let(:harvest_items) { create_harvest_items }
+
+  before do
+    # format the harvest items into a string of values for insertion
+    harvest_values = ''
+    harvest_items.each.with_index(1) do |example, index|
+      delimit = index == harvest_items.length ? '' : ",\n"
+      harvest_values += "(#{example})#{delimit}"
+    end
```

Comment:
didn't like this suggestion? https://github.com/QutEcoacoustics/baw-server/pull/718#discussion_r1959215908

Reply:
whoops, I missed that. i like it, will commit